packages/core/src/error-reporter.ts

```diff
@@ -38,6 +37,10 @@ export class ErrorReporter {
 		}
 	}
 
+	async shutdown(code = 1000) {
+		await close(code);
```

Comment:
`close` takes a timeout in milliseconds for flushing any pending error reports instead of a code

```suggestion
	async shutdown(timeoutInMs = 1000) {
		await close(timeoutInMs);
```